doc/tutorials/xfconf.md

```diff
@@ -0,0 +1,146 @@
+# Using Xfconf with Elektra
+
+Elektra provides the ability to work hand in hand with Xfconf.
+Depending on what you want, you can use Elektra in different ways.
+
+## Altering Existing Xfconf Settings
+
+Elektra provides an Xfconf storage plugin that allows you to change all the configuration settings made by Xfconf.
+
+To access the properties which are owned by Xfconf, the appropriate channel must be mounted.
+Some known channel names are
+
+- **Thunar**: thunar
+- **Xfwm**: xfwm4
+- **Xfce Panel**: xfce4-panel
+
+Assuming that the properties of Thunar are to be changed, the following command can be used to mount the channel.
+
+```shell
+kdb mount -R noresolver none /sw/xfce4/thunar xfconf channel=thunar
+```
+
+**Warning**: The following operations will cause permanent changes to your system, please handle with care.
+
+Thunar should now be ready to be configured by Elektra.
+For example, the following command can be used to use single clicks instead of double clicks.
+
+```shell
+# Setting the value to `false` beforehand
+xfconf-query -c thunar -p /misc-single-click -s false -t bool -n
+
+kdb set /sw/xfce4/thunar/misc-single-click TRUE
```

Comment:
Is `dir:` where this should end up? Maybe `user:` or spec might make more sense, no?

Reply:
Thank you for your review! You are right, the namespace which makes most sense to me are `user:` and `system:`. I just have overseen, that the default seemed to change.